events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
+
+### Requirements
+* Review this document.
+* A KCS Comms Shadow must either have, or plan to pursue, [Kubernetes Org Membership].
+* Commit to contributing to the KCS Staff team for multiple cycles.
+
+## Tasks
+During the course of their KCS cycle as a shadow, KCS Comms Shadows are expected to complete tasks including but not limited to:
+* Joining the appropriate Slack channels for KCS staff collaboration and for sending comms about KCS to the community
+* Working with the Comms Lead and Contributor Comms Subproject to get access to tooling for posting on Kubernetes Contributor-focused social media channels.
+* Representing the KCS Comms team in KCS Staff meetings.
+* Writing email comms as instructed by the KCS Comms Lead.
+* Writing social media comms as instructed by the KCS Comms Lead.
+* Reviewing comms written by other shadows, volunteers, or the lead as directed by the KCS Comms Lead
+
+Shadows should work with the Comms Lead to ensure they are getting sufficient experience to feel comfortable taking on the KCS Comms Lead role in a future cycle after completion of a shadow cycle.
+
+In the event a shadow is unable to complete the tasks expected of them as designated by the KCS Comms Lead, they should inform the KCS Comms Lead as early as possible, to the best of their ability. Shadows who are unable to complete the required tasks as designated by the KCS Comms Lead, will be removed from the list of shadows. Upon failure to complete a shadowing cycle, the contributor may request to become a shadow in a future cycle. Successful completion of a shadowing cycle (as determined by the KCS Comms Lead) or equivalent experience is required to become a KCS Comms Lead.
+
+## KCS Comms Volunteer Overview
+Volunteers are expected to assist with comms during the current KCS cycle, as directed by the lead. Volunteers do not need to have or be pursuing [Kubernetes Org Membership].
+
+### Requirements
+* Review this document.
+* Commit to supporting KCS Comms through the current cycle
+
+### Tasks
+During the course of their KCS cycle as a volunteer, KCS Comms Volunteers are expected to complete tasks including but not limited to:
+* Joining the appropriate Slack channels for KCS staff collaboration and for sending comms about KCS to the community
+* Working with the Comms Lead and Contributor Comms Subproject to get access to tooling for posting on Kubernetes Contributor-focused social media channels.
+* Writing email comms as instructed by the KCS Comms Lead.
+* Writing social media comms as instructed by the KCS Comms Lead.
+* [optional] Reviewing comms written by other volunteers, shadows, or the lead as directed by the KCS Comms Lead
+
+## KCS Website Manager
```

Comment:
This new role is a GREAT idea.

I think it's possible that non-KCS planning members could help here too (case in point, folks with Hugo and/or Docsy experience).

Might even be worth it to say they can lean on the Contributor Comms team for assistance too, as needed.